### PR TITLE
Created delete functionality

### PR DESCRIPTION
**Pull request by msaivasanth**
Created delete functionality for the website, open for review

### CODE REVIEW

File: src/ProductsContext/ProductState.tsx [hunk 1]
```diff
@@ -131,7 +131,24 @@ const ProductState: React.FC<ProductStateProps> = (props: any) => {
...
+      if(id != 0 || id < 100) {
+        const response = await fetch(`${host}/products/${id}`, {
+          method: 'DELETE' 
+        })
+        const json = await response.json()
+      }
+      
+    }
+  }
+  const value: any = { getProducts, products, setName, setPassword, name, password, handleLogin, handleGetDetails, handleAddItem, loading, handleDelete };
```
Comment (syedzeeshan1) [inline src/ProductsContext/ProductState.tsx]: do not use `any`. Create types for all, if you have it the same way in other locations, replace those too.

**Review by syedzeeshan1 — CHANGES_REQUESTED** [review]
Add a confirmation popup, maybe as a reusable directive and show success/failed messages on completion.

File: src/ProductsContext/ProductState.tsx (hunk 1 above)
Comment (msaivasanth) [reply src/ProductsContext/ProductState.tsx]: Made the changes in the code, please review

**Review by syedzeeshan1 — APPROVED** [review]
Great work!

**Comment by msaivasanth** [comment]
> Great work!

Thank you sir, Can I merge this branch with main.

**Comment by msaivasanth** [comment]
Adding delete functionality successfully

**Pull request merged**